Add flag to not target elements directly for typography.

Garbee opened the pull request:
Should *REALLY* not be used. However, if developers have a need for MDL without touching any base elements for typography, this will allow for that.

Maybe another variation (ugh) to add for the CDN via issue #1037. However, our CDN builds are already numerous, I'd be worried about adding yet another variation on top to add even more files. Still something to think about if we want to support it.

addyosmani commented:
> However, our CDN builds are already numerous, I'd be worried about adding yet another variation on top to add even more files. Still something to think about if we want to support it.

I'd have concerns about supporting this on the CDN (until we get asked about it a bunch), but I'm fine with the build supporting this otherwise.

addyosmani commented:
One very minor nit, otherwise lgtm :+1:

Garbee commented:
Updated comment to directly point out it is for typography. While I was in there, added a little extra to explain other components may break silently. Then also urged that the value not be touched. That will hopefully ward off anyone doing what some guide tells them blindly to turn it off.

addyosmani commented:
Thanks @Garbee!

thealjey commented:
I wish this was done with a css class instead (localized to a specific MDL component, not the whole document), because I'm not compiling sass currently and do not want to do it just to be able to use this library.
Global styles are not generally not a very good thing, and MDL is taking it way too far.
Also, pixel units, seriously?
